istioctl/cmd/analyze.go

```diff
@@ -102,6 +105,13 @@ istioctl analyze a.yaml b.yaml
 # Analyze yaml files without connecting to a live cluster
 istioctl analyze --use-kube=false a.yaml b.yaml
 
+# Analyze the current live cluster and suppress PodMissingProxy for pod mypod in namespace 'testing'.
+istioctl analyze -k -S "IST0103=Pod mypod.testing"
+
+# Analyze the current live cluster and suppress PodMissingProxy for all pods in namespace 'testing',
+# and suppress MisplacedAnnotation on deployment foobar in namespace default.
+istioctl analyze -k -S "IST0103=Pod *.testing" -S "IST0107=Deployment foobar.default"
```

Comment:
-k is now true by default, you can leave it off these examples.

Reply:
Thanks, fixed.